packages/core/src/config.ts

```diff
@@ -10,6 +10,7 @@ import {
   rspack,
 } from '@rsbuild/core';
 import glob from 'fast-glob';
+import ts from 'typescript';
```

Comment:
this will break JS projects

Reply:
We could use tsconfck to look up tsconfig, which is also used by Vite.

Reply:
I use `tsconfck` instead,  more details can be implemented when dealing no config feature.